src/t2iapi/device/service.proto

```diff
@@ -222,4 +222,27 @@ service DeviceService {
     */
     rpc InsertContainmentTreeEntryForSequenceId (InsertContainmentTreeEntryForSequenceIdRequest)
         returns (InsertContainmentTreeEntryForSequenceIdResponse);
+
+    /*
+    Determine whether the given descriptor represents a REMOVABLE SUBSYSTEM in the sense of
+    IEEE Std 11073-10700-2022.
+    */
+    rpc DoesDescriptorRepresentARemovableSubsystem(BasicHandleRequest)
+        returns (TrueFalseResponse);
+
+    /*
+    Physically disconnect the REMOVABLE SUBSYSTEM represented by the given descriptor. Ensure that it stays disconnected
```

Comment:
please use this text instead  (we are using it also for other manipulations)

> The manipulated state shall be persistent until a next manipulation call. If the device is not able to maintain
>   the static state, it shall return RESULT_NOT_SUPPORTED.

Reply:
Done.